src/Libs/OwlCore.WinUI/Controls/MarqueeTextBlock/MarqueeTextBlock.cs

```diff
@@ -364,7 +364,7 @@ private void StartMarqueeAnimationIfNeeded(bool useTransitions = true)
             }
         }
 
-        private void StoryboardMarquee_Completed(object sender, object e)
+        private void StoryboardMarquee_Completed(object? sender, object e)
```

Comment:
Wait why the sender is nullable, are we manually calling these event methods and passing null intentionally? 😄

Reply:
It's by contract in .NET 7. This object was always nullable, that's reflected now by the nullability annotations on `EventHandler`.